libs/application/templates/institution-collaboration/src/fields/Constraints/Constraints.tsx

```diff
@@ -20,162 +20,142 @@ const Constraints: FC<FieldBaseProps> = ({ field, application }) => {
-              m.constraints.constraintsLoginPlaceholder,
-              application,
-              formatMessage,
-            )}
-            defaultValue={getConstraintVal('hasLogin')}
-            extraText={false}
-          />
-          <ConstraintController
-            id={`${id}.straumur`}
-            checkboxId={`${id}.hasStraumur`}
-            label={formatText(
-              m.constraints.constraintsStraumurLabel,
-              application,
-              formatMessage,
-            )}
-            placeholder={formatText(
-              m.constraints.constraintsStraumurPlaceholder,
-              application,
-              formatMessage,
-            )}
-            defaultValue={getConstraintVal('hasStraumur')}
-            extraText={false}
-          />
-          <ConstraintController
-            id={`${id}.website`}
-            checkboxId={`${id}.hasWebsite`}
-            label={formatText(
-              m.constraints.constraintsWebsiteLabel,
-              application,
-              formatMessage,
-            )}
-            placeholder={formatText(
-              m.constraints.constraintsWebsitePlaceholder,
-              application,
-              formatMessage,
-            )}
-            defaultValue={getConstraintVal('hasWebsite')}
-            extraText={false}
-          />
-          <ConstraintController
-            id={`${id}.apply`}
-            checkboxId={`${id}.hasApply`}
-            label={formatText(
-              m.constraints.constraintsApplyingLabel,
-              application,
-              formatMessage,
-            )}
-            placeholder={formatText(
-              m.constraints.constraintsApplyingPlaceholder,
-              application,
-              formatMessage,
-            )}
-            defaultValue={getConstraintVal('hasApply')}
-            extraText={true}
-          />
+    <Box>
+      <Stack space={2}>
+        <ConstraintController
+          id={`${id}.mail`}
+          checkboxId={`${id}.hasMail`}
+          label={formatText(
+            m.constraints.constraintsMailLabel,
+            application,
+            formatMessage,
+          )}
+          placeholder={formatText(
+            m.constraints.constraintsMailPlaceholder,
+            application,
+            formatMessage,
+          )}
+          defaultValue={getConstraintVal('hasMail')}
+          extraText={false}
+        />
+        <ConstraintController
+          id={`${id}.login`}
+          checkboxId={`${id}.hasLogin`}
+          label={formatText(
+            m.constraints.constraintsLoginLabel,
+            application,
+            formatMessage,
+          )}
+          placeholder={formatText(
+            m.constraints.constraintsLoginPlaceholder,
+            application,
+            formatMessage,
+          )}
+          defaultValue={getConstraintVal('hasLogin')}
+          extraText={false}
+        />
+        <ConstraintController
+          id={`${id}.straumur`}
```

Comment:
Could we use english here instead of icelandic, most of the code is in english as we have other devs that dont speak icelandic?